modules/bmo-editing-the-hostfirmwaresettings-resource.adoc

```diff
@@ -0,0 +1,39 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="editing-the-hostfirmwaresettings-resource_{context}"]
+= Editing the HostFirmwareSettings resource
+
+You can edit the `HostFirmwareSettings` of provisioned hosts.
+
+.Procedure
+
+. Get the list of hosts:
++
+[source,terminal]
+----
+$ oc get hfs -n openshift-machine-api
+----
+
+. Edit a host's `HostFirmwareSettings` resource:
++
+[source,terminal]
+----
+$ oc edit hfs -n openshift-machine-api/<host_name>
```

Comment:
Please, change the command to `oc edit hfs <host_name> -n openshift-machine-api`. In the format that is given here the user will receive an error like `error: invalid namespace "openshift-machine-api/openshift-worker-0-1": [may not contain '/']`